oem/ibm/libpldmresponder/file_io.cpp

```diff
@@ -939,15 +940,38 @@ Response Handler::writeFileByTypeFromMemory(const pldm_msg* request,
 {
     return rwFileByTypeIntoMemory(PLDM_WRITE_FILE_BY_TYPE_FROM_MEMORY, request,
                                   payloadLength, oemPlatformHandler,
-                                  sharedAIORespDataobj);
+                                  sharedAIORespDataobj, instanceIdDb);
 }
 
 Response Handler::readFileByTypeIntoMemory(const pldm_msg* request,
                                            size_t payloadLength)
 {
     return rwFileByTypeIntoMemory(PLDM_READ_FILE_BY_TYPE_INTO_MEMORY, request,
                                   payloadLength, oemPlatformHandler,
-                                  sharedAIORespDataobj);
+                                  sharedAIORespDataobj, instanceIdDb);
+}
+
+void Handler::postWriteCallBack(
+    const uint16_t fileType, const uint32_t fileHandle,
```

Comment:
can we pass these params as a reference?

Reply:
Done